client/my-sites/importer/logos/godaddy-gocentral.jsx

```diff
@@ -5,11 +5,14 @@
  */
 import React from 'react';
 
-export default function GoDaddyGoCentralLogo( { size } ) {
+export default function GoDaddyGoCentralLogo( {
+	size,
+	className = 'godaddy-gocentral social-logo importer__service-icon',
+} ) {
 	return (
 		<img
 			// eslint-disable-next-line wpcalypso/jsx-classname-namespace
-			className="godaddy-gocentral social-logo importer__service-icon"
+			className={ className }
```

Comment:
Why this change?

Reply:
Please see response in `importer-logo.jsx`.